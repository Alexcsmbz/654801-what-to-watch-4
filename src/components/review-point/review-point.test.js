import ReviewPoint from './review-point.tsx';

describe(`ReviewPoint snapshot test`, () => {
  it(`ReviewPoint should render review point`, () => {
    const tree = renderer.create(
        <ReviewPoint
          onClick={() => {}}
          key="key"
          value={5}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
