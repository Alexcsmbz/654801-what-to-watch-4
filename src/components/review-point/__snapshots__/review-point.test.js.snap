// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewPoint snapshot test ReviewPoint should render review point 1`] = `
Array [
  <input
    className="rating__input"
    id="star-5"
    name="rating"
    type="radio"
    value={5}
  />,
  <label
    className="rating__label"
    htmlFor="star-5"
    onClick={[Function]}
  >
    Rating 
    5
  </label>,
]
`;
